src/RemoteAccessManager.php

```diff
@@ -87,7 +87,7 @@ private function isAllowed(): bool
 		return $allow;
 	}
 
-	/******************* interface \SixtyEightPublishers\RemoteAccessManager\IRemoteAccessManager *******************/
+	/******************** interface \SixtyEightPublishers\RemoteAccessManager\IRemoteAccessManager ********************/
```

Comment:
ehm. why? :smile:

Reply:
All these "interface comments" are long exactly 120 characters, this actually was 118.

Reply:
ah ok.